django/test/runner.py

```diff
@@ -540,11 +547,10 @@ class DiscoverRunner:
 
     def __init__(self, pattern=None, top_level=None, verbosity=1,
                  interactive=True, failfast=False, keepdb=False,
-                 reverse=False, debug_mode=False, debug_sql=False, parallel=0,
+                 reverse=False, debug_mode=False, debug_sql=False, parallel=1,
```

Comment:
Do we need to change the default value? IMO it's unnecessary. Also, this class is [documented](https://docs.djangoproject.com/en/3.2/topics/testing/advanced/#django.test.runner.DiscoverRunner).

Reply:
`0` is a weird value for `parallel`. It "accidentally" works - `--parallel` defaults to 1, but when 0 is passed, it acts the same as 1, since the condition for activating parallel testing is `if self.parallel > 1:`. 

I changed this since I thought it would make sense to align the class argument with the command line argument. If you really think it's unnecessary we can revert.

Reply:
We have the same consistent defaults in `setup_databases()` and `teardown_databases()`. It's just unrelated and backward incompatible. We can discuss this separately, but personally I think it's not worth changing due to concerns about backward incompatibility.

Reply:
Ok fair, reverting.